src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/AdjudicationMigrateDto.kt

```diff
@@ -113,6 +116,25 @@ data class MigrateEvidence(
   val reporter: String,
 )
 
+@Schema(description = "punishment / sanction")
+data class MigratePunishment(
+  @Schema(description = "sanction code")
+  val sanctionCode: String,
+  @Schema(description = "sanction status")
+  val sanctionStatus: String,
+  @Schema(description = "sanction sequence")
+  val sanctionSeq: Long,
+  @Schema(description = "effective date")
+  val effectiveDate: LocalDateTime,
+  @Schema(description = "comment text")
+  val comment: String?,
+  @Schema(description = "compensation amount")
+  val compensationAmount: BigDecimal?,
+  @Schema(description = "consecutive charge number - for nomis this needs to be the oic_incident_id for the consecutiveOffenderBookId/consecutiveSanctionSeq")
+  val consecutiveChargeNumber: Long?,
```

Comment:
i dont think we need the consecutiveChargeSeq, but you will need to map the details to the relevant oic_incident_id for us